src/services/guardrail_service.py

```
@@ -0,0 +1,277 @@
+        return self._langfuse
+
+    def _resolve_ollama_endpoint(self) -> str:
+        try:
+            from config.settings import config_manager
+            from utils.container_utils import transform_localhost_url
+
+            cfg = config_manager.get_config()
+            if cfg and cfg.providers.ollama.endpoint:
+                return transform_localhost_url(cfg.providers.ollama.endpoint)
+        except Exception:
+            pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)